### DISCUSSION TITLE
visualize non-human kegg pathways

### DISCUSSION THREAD

**Discussion by aswathyseb** [egeulgen/pathfindR#41] [Ideas]
Hi,

Thank you for the details on using pathfindR for non-human organisms.
Is there a way to visualize KEGG pathways for non-human organisms.

In the example using mouse, I found we can visualize enriched terms, but am not clear if I can visualize mouse KEGG pathways.

Could you please let me know?
Thanks,

**Comment by egeulgen** [comment]
hello,

There is currently no implemented functionality to visualize non-hsa KEGG pathway diagrams. You may try to visualize through [pathview](https://bioconductor.org/packages/release/bioc/html/pathview.html)
hope this helps,
-E
